addons/website/static/src/scss/website.wysiwyg.scss

```diff
@@ -59,12 +59,10 @@ html[data-edit_translations="1"] {
     &::before {
         position: absolute;
         // Content is 0px wide => use available width.
-        width: -webkit-fill-available;
-        width: -moz-available;
         right: 20px;
         z-index: 100;
         background-color: $o-we-accent;
-        font-size: 0px;
+        font-size: 200px;
```

Comment:
Instead touching the font size, I prefer to introduce width attribute along with `--webkit-mask`.
```
    &.d-md-none::before, &.d-lg-none::before {
        height: 50px;
        width: 50px;
        -webkit-mask: url("/website/static/src/img/snippets_options/desktop_invisible.svg") no-repeat 100% 100%;
    }
```

there is possibly no way width extends more that 50px(of height) in this particular case.

Reply:
@divy-odoo Good to go. Thanks for review!